packages/react/src/components/AccountSettings/ChangePassword/__tests__/__snapshots__/ChangePassword.test.tsx.snap

```diff
@@ -43,7 +41,6 @@ exports[`ChangePassword renders as expected 1`] = `
               aria-checked="false"
               aria-label="Show password"
               class="amplify-button amplify-field-group__control amplify-field__show-password"
-              data-fullwidth="false"
```

Comment:
This isn't used either?

Reply:
Correct, doing a search in the repo for `data-fullwidth` only comes up with snapshot tests and vue/angular code. No CSS references it.